packages/react/src/global.js

```diff
@@ -136,3 +136,7 @@ export let Global: React.AbstractComponent<
 
   return null
 })
+
+if (process.env.NODE_ENV !== 'production') {
+  Global.displayName = 'EmotionGlobal'
```

Comment:
Same as here: https://github.com/emotion-js/emotion/pull/2132/files#r530285457

Please note that I'm open for discussion about this.

Reply:
I prefixed it with Emotion for following reasons:
* It shows up when I search for `emotion` in React devtools which might help me find emotion-related components.
* It is more specific than `Global`. If someone new to a codebase sees this component and search for `EmotionGlobal` on Google, they find the correct docs immediately.
* It doesn't seem to have any disadvantages. When I search for `Global`, I still find this component.

But I don't have a big opinion on this. I'm happy to change them to their exact name if you think it makes more sense.